packages/mobile/src/firebase/notifications.ts

```diff
@@ -87,27 +87,22 @@ export function* handleNotification(
   message: FirebaseMessagingTypes.RemoteMessage,
   notificationState: NotificationReceiveState
 ) {
-  // See if this is a notification with an open url action (`ou` prop in the data)
+  // See if this is a notification with an open url or webview action (`ou` or `ow` prop in the data)
   const urlToOpen = message.data?.ou
+  const webViewToOpen = message.data?.ow
```

Comment:
This works, but what do you think of making it the default behavior to open it in a webview and providing another parameter to explicitly disable the webview handling? for instance (`openExternal: true`)
This would save us from duplicating the passed url and we would only use the extra parameter if really necessary.

Reply:
Sounds good, made WebView the default :)